velox/type/StringView.h

```diff
@@ -77,6 +78,12 @@ struct StringView {
     }
   }
 
+  void makeInline(uint32_t size, char*& buffer) {
```

Comment:
adding the second parameter makes this API a bit more convoluted. Maybe we can keep it simple and expect clients to just:

```
sv.makeInline(4);
auto ptr = sv.data();
```

?

Reply:
The intent here was to prevent anyone needing to `const_cast` the data from a string view. I'll use your suggestion below and write `static StringView StringView::makeInline(std::string)`